client/src/Tools/_framework/ToolPanels/MoveContentToFolder.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useState } from "react";
+import {
+  Text,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  SimpleGrid,
+  Heading,
+  IconButton,
+  HStack,
+} from "@chakra-ui/react";
+import axios from "axios";
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import { useFetcher } from "react-router-dom";
+import { ActivityStructure } from "../Paths/ActivityEditor";
+
+interface activeView {
```

Comment:
Use PascalCase for interfaces and types. Also, this should probably be a type.

Reply:
Done. Now it's a type named ActiveView.